gandicli.man.rst

```diff
@@ -228,6 +229,8 @@ Details:
 
 * ``gandi domain create domain.tld`` helps register a domain. Options are ``--domain domain.tld`` for the domain you want to get (/!\ this option is deprecated and will be removed upon next release), ``--duration INTEGER RANGE`` for the registration period, ``--owner TEXT``, ``--admin TEXT``, ``--tech TEXT``, ``--bill TEXT`` for the four contacts to pass to the creation process, ``--nameserver TEXT`` for adding custom nameservers. All these modification can be done as background process using the option ``--background`` (or ``--bg``).
 
+* ``gandi  domain check omain.tld`` Check information about the specific domain availability and price.
```

Comment:
typo in the command: should be `domain.tld`.
And it would be more explicit to say `Retrieve information..` than `Check information...`